.github/workflows/publish-algorithm-images.yaml

```diff
@@ -26,6 +26,7 @@ jobs:
           dockerfile: ${{ matrix.dockerfile }}
 
     strategy:
+      fail-fast: false
```

Comment:
Is this added to see all errors at once?

Reply:
Yes.